### PR TITLE
Deprecation

### PR DESCRIPTION
**Pull request by iteloo**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -7,7 +7,7 @@ jobs:
     docker:
       # specify the version
       - image: colinmerkel/mushu-ci:latest
-      
+
```
Comment (colin353) [inline .circleci/config.yml]: Why change formatting?

File: .circleci/config.yml [hunk 2]
```diff
@@ -16,8 +16,6 @@ jobs:
       - run: cd server; go get -v -t -d ./...
       - run: cd server; go test -v ./...
 
-        # Make sure the formatting is right.
-      - run: cd js; sysconfcpus -n 2 elm-format --validate *.elm
```
Comment (colin353) [inline .circleci/config.yml]: We can delete this altogether, if you want. I don't necessarily care about format being enforced.

**Review by colin353 — COMMENTED** [review]
LGTM.

File: .circleci/config.yml (hunk 1 above)
Comment (iteloo) [reply .circleci/config.yml]: Atom automatically does that on save, sorry.

File: .circleci/config.yml (hunk 2 above)
Comment (iteloo) [reply .circleci/config.yml]: Sure

**Review by colin353 — APPROVED** [review]
LGTM

**Pull request merged**